ouroboros-consensus-test/test-consensus/Test/Consensus/Mempool.hs

```diff
@@ -77,6 +92,217 @@ tests = testGroup "Mempool"
   , testProperty "removeTxs"                               prop_Mempool_removeTxs
   ]
 
+{-------------------------------------------------------------------------------
+  Mempool Pure Properties
+-------------------------------------------------------------------------------}
+
+-- | Test that removed transactions are not in the
+--   result transactionsIds of the returned internal state.
```

Comment:
```suggestion
-- result transactionsIds of the returned internal state.
```
Also, there's something wrong with "are not in the result transactionsIds of the returned internal state". Something like "Test that after removing the transactions the Mempool no longer contains those transactions"?